packages/relay/src/lib/clients/mirrorNodeClient.ts

```diff
@@ -297,6 +304,9 @@ export class MirrorNodeClient {
         const acceptedErrorResponses = MirrorNodeClient.acceptedErrorStatusesResponsePerRequestPathMap.get(pathLabel);
         if (error.response && acceptedErrorResponses && acceptedErrorResponses.indexOf(effectiveStatusCode) !== -1) {
             this.logger.debug(`${requestIdPrefix} [${method}] ${path} ${effectiveStatusCode} status`);
+            if(pathLabel  === MirrorNodeClient.CONTRACT_CALL_ENDPOINT) {
```

Comment:
I think you should add a new set of tests for `handleError()`
This can confirm for `contract/call` that the codes we allow return null and the ones we don't throw the error

Reply:
Maybe
```ts
describe('handleError', async() => {

    const CONTRACT_CALL_ENDPOINT = 'contracts/call';
    const nullResponseCodes = [400,404,415,429,500];
    const errorRepsonseCodes = [501, 503];

    for (const code of nullResponseCodes) {
      it(`returns null when ${code} is returned`, async () => {
        const result = await mirrorNodeInstance.handleError(new Error('test error'), CONTRACT_CALL_ENDPOINT, code, 'POST');
        expect(result).to.equal(null);
      });
    }

    for (const code of errorRepsonseCodes) {
      it(`throws an error when ${code} is returned`, async () => {
        try {
          await mirrorNodeInstance.handleError(new Error('test error'), CONTRACT_CALL_ENDPOINT, code, 'POST');
          expect.fail('should have thrown an error');
        } catch (e) {
          expect(e.message).to.equal('test error');
        }
      });
    }
  });
```

Reply:
Thank for  the  starting point,  good suggestion!  👍 

✅   done